File: app/src/main/java/com/bkjcb/rqapplication/check/model/CheckResultItem.java
package com.bkjcb.rqapplication.check.model;

import com.bkjcb.rqapplication.datebase.ObjectBox;

import io.objectbox.Box;
import io.objectbox.annotation.Entity;
import io.objectbox.annotation.Id;

/**
 * Created by DengShuai on 2019/12/24.
 * Description :
 */
@Entity
public class CheckResultItem {
    @Id(assignable = true)
    public long id;
    public String jianchajilu;
    public String jianchaid;
    public String jianchaxiangid;

    public CheckResultItem() {
    }

    public CheckResultItem(String jianchaid, String jianchaxiangid) {
        this.jianchaid = jianchaid;
        this.jianchaxiangid = jianchaxiangid;
    }

    public static Box<CheckResultItem> getBox(){
        return ObjectBox.INSTANCE.getBoxStore().boxFor(CheckResultItem.class);
    }
}

File: app/src/main/java/com/bkjcb/rqapplication/check/model/CheckResultItem.kt
package com.bkjcb.rqapplication.check.model

import io.objectbox.annotation.Entity
import io.objectbox.annotation.Id

/**
 * Created by DengShuai on 2019/12/24.
 * Description :
 */
@Entity
class CheckResultItem() {
    @Id(assignable = true)
    var id: Long = 0
    var jianchajilu: String? = null
    var jianchaid: String? = null
    var jianchaxiangid: String? = null

    constructor(jianchaid: String, jianchaxiangid: String) : this() {
        this.jianchaid = jianchaid
        this.jianchaxiangid = jianchaxiangid
    }
}